### ISSUE TITLE
question about Eq. (7) 

### ISSUE THREAD

**Issue by zzowenzz** [Zhaozixiang1228/MMIF-DDFM#25]
Hi @Zhaozixiang1228, excellent paper and I appreciate for open-souring it. I would like to ask a question about the equation (7), which initially is 
$\nabla_{f_t}\log p_t(f_t|i,v) = \nabla_{f_t}\log p_t(f_t) + \nabla_{f_t}\log p_t(i,v|f_t)$
For the second part on the right side of the equation $\nabla_{f_t}\log p_t(i,v|f_t)$, which is the likelihood that probably to assess how good the fusion process is. In my mind, it serves somehow like the loss function used in image fusion. Therefore, from the perspective of the loss function, will it make the overall fusion better by fine-tuning the model? I understand that inference-only is awesome, but I would like to know if what I'm saying is even feasible.
